packages/number-field/test/number-field.test.ts

```diff
@@ -117,6 +117,40 @@ describe('NumberField', () => {
             expect(el.focusElement.value).to.equal('13 377 331');
         });
     });
+    xit('correctly interprets decimal point on iPhone', async () => {
```

Comment:
I created a test with some common usecases but haven't been able to run it since I couldn't find a way to mock isIphone.